src/utilities/fetch-supporters.js

```diff
@@ -9,9 +9,24 @@ const asyncWriteFile = promisify(fs.writeFile);
 
 const REQUIRED_KEYS = ['totalDonations', 'slug', 'name'];
 const filename = '_supporters.json';
-const absoluteFilename = path.resolve(__dirname, '..', 'components', 'Support', filename);
+const absoluteFilename = path.resolve(
+  __dirname,
+  '..',
+  'components',
+  'Support',
+  filename
+);
 
-const graphqlEndpoint = 'https://api.opencollective.com/graphql/v2';
+let graphqlEndpoint =
+  'https://api.opencollective.com/graphql/v2';
+
+// https://github.com/opencollective/opencollective-api/blob/master/server/graphql/v2/query/TransactionsQuery.ts#L81
+const graphqlPageSize = 1000;
+
+if (process.env && process.env.CI && process.env.OPENCOLLECTIVE_API_KEY) {
+  // use api key when deploying to production
```

Comment:
Currently I'm using a api key from my own opencollective account, you can replace it here https://github.com/webpack/webpack.js.org/settings/secrets/actions

Reply:
Do we really need an API key for that. That should be public data and it worked without before too.

Reply:
ok, i'll remove it and let's see how the improved version works first.